src/main.cpp

```diff
@@ -1,18 +1,27 @@
 #include "Server.hpp"
 #include "ConfigFile.hpp"
 
+# define CONFIG_FILE_DEFAULT_PATH "./config"
```

Comment:
The default config file of NGNIX is name `ngnix.conf`, the one of apache `apache2.conf` or `httpd.conf`. Should we not adapt the name of our config file accordingly, like `webserv.conf`?